auditors/privileged/privileged.go

```diff
@@ -0,0 +1,85 @@
+package privileged
+
+import (
+	"github.com/Shopify/kubeaudit"
+	"github.com/Shopify/kubeaudit/internal/k8s"
+	"github.com/Shopify/kubeaudit/internal/override"
+	"github.com/Shopify/kubeaudit/k8stypes"
+)
+
+const (
+	// PrivilegedTrue occurs when privileged is set to true in the container SecurityContext
+	PrivilegedTrue = "PrivilegedTrue"
+	// PrivilegedNil occurs when privileged is not set in the container SecurityContext.
+	// Prvivileged defaults to false so this is ok
+	PrivilegedNil = "PrivilegedNil"
+)
+
+const OverrideLabel = "allow-privileged"
+
+// Privileged implements Auditable
+type Privileged struct{}
+
+func New() *Privileged {
+	return &Privileged{}
+}
+
+// Audit checks that privileged is set to false in every container's security context
+func (a *Privileged) Audit(resource k8stypes.Resource, _ []k8stypes.Resource) ([]*kubeaudit.AuditResult, error) {
+	var auditResults []*kubeaudit.AuditResult
+
+	for _, container := range k8s.GetContainers(resource) {
+		auditResult := auditContainer(container, resource)
+		auditResult = override.ApplyOverride(auditResult, container.Name, resource, OverrideLabel)
+		if auditResult != nil {
+			auditResults = append(auditResults, auditResult)
+		}
+	}
+
+	return auditResults, nil
+}
+
+func auditContainer(container *k8stypes.ContainerV1, resource k8stypes.Resource) *kubeaudit.AuditResult {
+	if isPrivilegedNil(container) {
+		// TODO:  should "warning" results include a fix?
```

Comment:
Since the change her doesn't change functionality I think we should just include the fix.